### DISCUSSION TITLE
How to change light position?

### DISCUSSION THREAD

**Discussion by afif-fahim** [google/model-viewer#2351] [Q&A]
Currently there is a light at the top of the model. How to rotate or change the position of light?

**Comment by felipe-chamas** [comment]
You can't. The [environment is pre-built](https://github.com/google/model-viewer/blob/master/packages/model-viewer/src/three-components/EnvironmentScene.ts) on three-js before loading. There's a [more neutral](https://github.com/google/model-viewer/pull/2249) environment coming up, but if it's still not good for you I think the best option is setting the [environment-image](https://modelviewer.dev/docs/index.html#entrydocs-lightingandenv-attributes-environmentImage) from `jpg` or `hdr`.
